### PR TITLE
DT-14432 adding language field for Va Form Node template.

### PR DESCRIPTION
**Pull request by zacharymorel**
## Description
Per this conversation, we are adding the html attribute as the inter solution.
https://github.com/department-of-veterans-affairs/va.gov-team/issues/14432

## Testing done
Ran unit and e2e. Spun up preview server: 

## Screenshots
Screen Shot 2021-04-01 at 4 44 08 PM

## Acceptance criteria
- [x] lang attribute populates

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/layouts/va_form.drupal.liquid [hunk 1]
```diff
@@ -17,7 +17,11 @@
 
         <dl>
           <div class="vads-u-margin-bottom--4">
-            <dt class="va-introtext">
+            {%if fieldVaFormLanguage %}
+              <dt class="va-introtext" lang={{ fieldVaFormLanguage }}>
```
Comment (ncksllvn) [inline src/site/layouts/va_form.drupal.liquid:21]: Can we add the quotes around the lang?

```suggestion
              <dt class="va-introtext" lang="{{ fieldVaFormLanguage }}">
```

**Review by ncksllvn — APPROVED** [review]
LGTM!

File: src/site/layouts/va_form.drupal.liquid [hunk 2]
```diff
@@ -17,7 +17,11 @@
 
         <dl>
           <div class="vads-u-margin-bottom--4">
-            <dt class="va-introtext">
+            {%if fieldVaFormLanguage %}
```
Comment (ncksllvn) [inline src/site/layouts/va_form.drupal.liquid:20]: May need a space before the `if`

**Comment by ncksllvn** [comment]
Btw we have support for unit tests now for Liquid templates, in case you want to try that out. Her are the tests for the landing_page layout `src/site/layouts/tests/landing_page`

**Review by U-DON — APPROVED** [review]

**Pull request merged**